base/libgit2/rebase.jl

```diff
@@ -41,6 +41,11 @@ function Base.next(rb::GitRebase)
     return unsafe_load(rb_op_ptr_ptr[])
 end
 
+function Base.show(io::IO, rb::GitRebase)
+    println(io, "GitRebase:")
+    println(io, "Number of operations to apply: $(count(rb))")
```

Comment:
Could you just do `"Number ...: ", count(rb)` to avoid the interpolation? Might be very marginally faster